pkg/kubelet/rkt/rkt_test.go

```diff
@@ -937,24 +937,26 @@ func TestSetApp(t *testing.T) {
 	rootUser := int64(0)
 	nonRootUser := int64(42)
 	runAsNonRootTrue := true
-	fsgid := int64(3)
+	fsgid := int64(4)
```

Comment:
Wanted to match the append order i.e. when appending groups put fsgroup last, so I changed the test. I'll change the test back and change the implementation instead